src/components/signup/components/Signup.js

```diff
@@ -0,0 +1,237 @@
+                            })(
+                                <Input />
+                            )}
+                        </FormItem>
+                        <FormItem
+                            {...formItemLayout}
+                            label={LABEL_4}
+                        >
+                            {getFieldDecorator("title", {
+                                rules: [{ message: { MESSAGE_7 }, required: true }],
+                            })(
+                                <Input />
+                            )}
+                        </FormItem>
+                        <FormItem
+                            {...formItemLayout}
+                            label={PASSWORD}
+                        >
+                            {getFieldDecorator("password", {
+                                rules: [{
+                                    message: { MESSAGE_8 }, required: true,
+                                }, {
+                                    validator: this.validateToNextPassword,
+                                }],
+                            })(
+                                <Input type={PASSWORD} />
+                            )}
+                        </FormItem>
+                        <FormItem
+                            {...formItemLayout}
+                            label= {LABEL_5}
+                        >
+                            {getFieldDecorator("confirm", {
+                                rules: [{
+                                    message: { MESSAGE_9 }, required: true,
+                                }, {
+                                    validator: this.compareToFirstPassword,
+                                }],
+                            })(
+                                <Input type={PASSWORD} onBlur={this.handleConfirmBlur} />
+                            )}
+                        </FormItem>
+                        <FormItem 
+                            className={CLASSNAME_AGREEMENT}
+                        >
+                            {getFieldDecorator("agreement", {
+                                valuePropName: "checked",
+                            })(
+                                <Checkbox>
+                                    {READ} 
+                                    <a href="">
+                                        {AGREEMENT}
+                                    </a>
+                                </Checkbox>
+                            )}
+                        </FormItem>
+                        <FormItem>
+                            <Button
+                                type={PRIMARY}
+                                htmlType={SUBMIT}
+                                shape={CIRCLE}
+                                className={CLASSNAME_SIGNUP}
+                            > 
+                                {TITLE}
+                            </Button>
```

Comment:
delete this button, use the `okText` on the Modal instead
```
...
okText={TITLE}
...
```

Reply:
OK